File: messages/src/main/java/com/thundashop/core/trackandtrace/PooledDestionation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.trackandtrace;

import com.thundashop.core.common.DataCommon;
import org.mongodb.morphia.annotations.Transient;

/**
 *
 * @author ktonder
 */
public class PooledDestionation extends DataCommon {
    public String destionationId;
    public String pooledByUserId = "";
    public String originalRouteId = "";
    
    @Transient
    public Destination destination;
    
    @Transient
    public Route originalRoute;
}

File: html5apps/terminal/events/core/trackandtrace/PooledDestionation.php
<?php
class core_trackandtrace_PooledDestionation extends core_common_DataCommon  {
	/** @var String */
	public $destionationId;

	/** @var String */
	public $pooledByUserId;

	/** @var String */
	public $originalRouteId;

	/** @var core_trackandtrace_Destination */
	public $destination;

	/** @var core_trackandtrace_Route */
	public $originalRoute;

}
?>